include/pokemon.h

```diff
@@ -443,8 +444,13 @@ struct SpeciesInfo /*0x8C*/
  /* 0x84 */ const struct FormChange *formChangeTable;
 };
 
-struct BattleMove
+struct MoveInfo
 {
+    union {
```

Comment:
I should've asked this before but, why aren't we using compound string for all move names?

Reply:
Compiler errors in some parts of the codebase.

Reply:
I'll see if I can PR some help your way

Reply:
I thought it was due to the fact that bad alignment of compound strings meant that an array was more space efficient?

Reply:
^ Not for me. My first priority was to get something that worked, and after I got something that worked I focused on the other aspects such as actually porting the move names and such 😂

Reply:
Oh, well then if we can fix those bugs it probably is worth just using a compound string for everything :P We should probably run the numbers though to check.

Reply:
I thought Lunos (or someone else :thinking:) did run the numbers and it works out smaller to embed the names? It certainly seems like it should, we're talking 10 bytes per embedded name vs 4 bytes + (n+1) bytes for `COMPOUND_STRING`, where `n` is the length of the name. So it's pretty hard for that to get below 10 bytes unless we think most names are fewer than 6 letters long.

Reply:
I applied usage of only compound strings and ended up saving 1180 bytes.
I'm PRing to this branch those changes + description alignment in the same style as Pokédex entries.